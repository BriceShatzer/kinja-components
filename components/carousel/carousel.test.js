/* @flow */

import * as React from 'react';
import { shallow } from 'enzyme';
import posts from 'kinja-components/__stubs__/postObjects.json';
import Carousel from './carousel';

describe('<Carousel />', () => {
	it('should render by default', () => {
		const wrapper = shallow(<Carousel posts={posts.items.slice(0, 15)} />);
		expect(wrapper).toMatchSnapshot();
	});
});
