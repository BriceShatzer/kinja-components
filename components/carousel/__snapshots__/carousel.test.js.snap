// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Carousel /> should render by default 1`] = `
<carousel__Items>
  <Carousel
    afterSlide={[Function]}
    autoGenerateStyleTag={true}
    autoplay={false}
    autoplayInterval={3000}
    autoplayReverse={false}
    beforeSlide={[Function]}
    cellAlign="left"
    cellSpacing={32}
    disableAnimation={false}
    disableEdgeSwiping={true}
    dragging={true}
    easing="easeCircleOut"
    edgeEasing="linear"
    enableKeyboardControls={false}
    frameOverflow="normal"
    framePadding="0px"
    height="inherit"
    heightMode="max"
    initialSlideWidth={223}
    onDragStart={[Function]}
    onResize={[Function]}
    pauseOnHover={true}
    renderAnnounceSlideMessage={[Function]}
    renderBottomCenterControls={[Function]}
    renderCenterLeftControls={[Function]}
    renderCenterRightControls={[Function]}
    renderTopCenterControls={[Function]}
    slideHeight="auto"
    slideIndex={0}
    slideListMargin={10}
    slideOffset={25}
    slideWidth={1}
    slidesToScroll={4}
    slidesToShow={4}
    speed={300}
    style={Object {}}
    swiping={true}
    transitionMode="scroll"
    vertical={false}
    width="100%"
    withoutControls={false}
    wrapAround={false}
  />
</carousel__Items>
`;
